app/templates/components/widgets/forms/link-input.hbs

```diff
@@ -25,6 +31,60 @@
       {{/if}}
     </div>
   </div>
+{{else if (eq this.inputId 'speaker_twitter')}}
```

Comment:
`speaker_twitter` will be specefic to speaker form only, I think we can normazlize it like below. Can you try if something like below is working?

```suggestion
{{else if this.inputId.includes('twitter')}}
```

Reply:
Ok